### ISSUE TITLE
How about a health eating DIP?

### ISSUE THREAD

**Issue by charlesrich** [always-on/always#295]
@zessiez That's really what it sonds like :-).  Maybe reword or respell?

**Comment by zessie** [comment]
This is what we have in the script "How about a healthy eating tip?". Do you think we need to reword it?

**Comment by charlesrich** [comment]
@zessie Yes, listen to it and you will see what I mean.

**Comment by zessiez** [comment]
OK.

**Comment by zessie** [comment]
This has been fixed.
